### PR TITLE
Fix install of clock.tcl

### PR DESCRIPTION
**Pull request by bovine**
BCK-6471

### CODE REVIEW

**Review by resuna — COMMENTED** [review]
Is this the correct terminology? Doesn't "stubs" have a specific meaning in Tcl?

**Comment by bovine** [comment]
> Is this the correct terminology? Doesn't "stubs" have a specific meaning in Tcl?

That's a different meaning that you're thinking of. The use of the term "stubs" was pre-existing on this line: https://github.com/flightaware/tclclockmod/pull/9/files#diff-64aa0fa0a37b3b9d22830b15c8430837b36b0d095dcc65ee07fd22bab5353c05R47

**Comment by bovine** [comment]
Note that a side-effect of using `lib_BINARIES` in this way causes the clock.tcl to be installed within a nested `lib/` directory, however that seems to work fine since pkgIndex.tcl and loader.tcl both search for clock.tcl within an optional lib subdirectory.
```
# ls -lR /usr/lib/tclclockmod8.6.707/
/usr/lib/tclclockmod8.6.707/:
total 132
drwxr-xr-x 2 root root   4096 Oct 13 18:25 lib
-rw-r--r-- 1 root root 123112 Oct 13 18:25 libtclclockmod8.6.707.so
-rw-r--r-- 1 root root   1284 Oct 13 18:25 pkgIndex.tcl

/usr/lib/tclclockmod8.6.707/lib:
total 60
-rw-r--r-- 1 root root 60259 Oct 13 18:25 clock.tcl
```

**Comment by resuna** [comment]
Yes, I know that it's a different meaning, that's why I was confused. I think calling it the clock ensemble or Tcl library would be better.

Also, update configure.ac to 8.6.709?

File: lib/loader.tcl [hunk 1]
```diff
@@ -35,16 +35,16 @@ proc ::clock args {
 
 
   # first try from the lib directory (like installed):
-  # second try to find stubs in the same directory as the shared library.
+  # second try to find in the same directory as the shared library.
   set stubs [glob -nocomplain [file join $::tcl::clock::LibDir clock.tcl] \
```
Comment (resuna) [inline lib/loader.tcl]: The variable name is still stubs.

**Pull request merged**